web/documentserver-example/java-spring/src/main/java/com/onlyoffice/integration/documentserver/models/filemodel/User.java

```diff
@@ -39,7 +39,7 @@ public class User extends AbstractModel {
 
     // the user configuration parameters
     public void configure(final int idParam, final String nameParam, final String groupParam) {
-        this.id = "" + idParam;  // the user id
+        this.id = "uid-" + idParam;  // the user id
```

Comment:
I'm confused a bit because I don't understand why the prefix was removed in the first place and how you were able to easily bring it back without making any changes to your feature. Can you please explain why you initially chose to remove the prefix?